ecommerce/certprep/static/sass/base/_basket.scss

```diff
@@ -174,6 +176,14 @@ $basket-items-color: #ddd;
                     background: darken($brand-primary, 10%);
                 }
             }
+
+            .support-text {
+                position: absolute;
```

Comment:
Did you test this in mobile views? @sergivalero20

Reply:
Yes, I did @Squirrel18 

![image](https://user-images.githubusercontent.com/73655023/216675193-9b1f8c07-a189-483a-aa59-95baf4630cb5.png)